ocs_ci/ocs/managedservice.py

```diff
@@ -32,3 +42,55 @@ def get_dms_secret_name():
     return (
         config.DEPLOYMENT["addon_name"] + constants.MANAGED_DEADMANSSNITCH_SECRET_SUFFIX
     )
+
+
+def disable_odf_operator_and_update_pull_secret():
+    """
+    For unreleased odf operator version -  Disable odf operator
+      1. Disable odf operator
+      2. Update pull secrete
+      3. Create a catalogSource using the ocs-registry image
+
+    Note: This is a hack done to allow odf to odf deployment before full addon is available.
+    """
+    oc = ocp.OCP(kind=constants.SECRET, namespace="openshift-config")
+    logger.info("Disable odf operator")
+    exec_cmd(
+        'oc patch operatorhub.config.openshift.io/cluster -p=\'{"spec":{"sources":'
+        '[{"disabled":true,"name":"redhat-operators"}]}}\' --type=merge'
+    )
```

Comment:
See function [`disable_specific_source(source_name)`](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/catalog_source.py#L149) - it might be reused here I think.